metadata.json

```
@@ -9,6 +9,9 @@
     "requires": {
         ">=2.0.1": [
```

```suggestion
        ">=2.0.1,<2.0.3": [
```
In this way it is non-ambiguous.

---

Okay. And this addition is required and cannot be deduced automatically by the system from the current setup?
I'm automating this part, hence I'd like to have as few edits for it as possible 😅

---

No, while the implied prevalence may seem obvious in simple cases like this, there is no way to automatically deduce which constraint should take prevalence in general. However, exactly for this reason, we'll revise the app store, to reduce/eliminate the need to manually specify constraints like this. Will discuss at today's AiiDAlab meeting.

---

Then I'll wait slightly with updating this until after the meeting.